lib/tx_build/signature.ex

```diff
@@ -60,6 +52,21 @@ defmodule Stellar.TxBuild.Signature do
     |> DecoratedSignature.new(signature)
   end
 
+  @spec build_signature(public_key :: String.t(), secret :: String.t()) :: t()
+  def build_signature(public_key, secret) do
```

Comment:
Is this a public function?

Reply:
nope, it is not! just updated it. Thank you!